app/assets/stylesheets/farmbot.css.scss

```diff
@@ -98,6 +101,46 @@ $brown:          #CA8;
   }
 }
 
+.designer-crop-info {
+  .crop-drag-info-image {
+    max-width: 98%;
+    padding: 0 auto 0 auto;
+    background-color: $translucent;
+  }
+  .crop-info-overlay {
+    position: relative;
+    bottom: 3rem;
+    z-index: 3;
+    background-color: $dark_gray;
+    font-style: italic;
+    color: $white;
+    display: flex;
+    justify-content: center; /* align horizontal */
+    align-items: center; /* align vertical */
+    height: 3rem;
+    max-width: 98%;
+  }
+  padding: 0 0 0 5px;
+  color: $dark_gray;
+  a {
+    color: $dark_gray;
+    padding: 0 0 0 4px;
+    text-transform: none;
+    text-decoration: underline;
+    font-weight: normal;
+    font-size: 0.75rem;
+  }
+  h6 {
+    font-size: 0.8rem;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+}
+
+.plantCatalogTile{
```

Comment:
Selector `plantCatalogTile` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space